### ISSUE TITLE
Render sky error

### ISSUE THREAD

**Issue by Faked4075** [wisp-forest/affinity#29]
See this crash report please, can i fix?
[crash-2024-12-02_17.16.04-client.txt](https://github.com/user-attachments/files/17977097/crash-2024-12-02_17.16.04-client.txt)

**Comment by Faked4075** [comment]
> From reading the crash report it seems like PojavLauncher has some custom OpenGL implementation, which might be causing this crash. Generally we would need more info about this crash like:
> 
> * Does this happen with only Affinity installed?
> * Does this happen exclusively on Android (with Pojav?)
>   If these points are the case, then this issue might need to be reported to Pojav

yes, it happens only with affinity installed, maybe there are ways to get rid of problematic capturesky? please.

**Comment by gliscowo** [comment]
I'm afraid I cannot reproduce this issue with just the mods required to run Affinity and stock Pojav settings. Seeing as you appear to be running on an OpenGL 3 emulation layer, I fear your phone may simply be too old to support the OpenGL features necessary to run Affinity. Alternatively you might simply need to switch to the Holy GL4ES renderer, with which I confirmed that Affinity works just fine

Cheers
